packages/plugin-ext/src/main/browser/output-channel-registry-main.ts

```diff
@@ -14,21 +14,25 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 import {interfaces} from 'inversify';
-import {OUTPUT_WIDGET_KIND} from '@theia/output/lib/browser/output-widget';
+import {OutputWidget} from '@theia/output/lib/browser/output-widget';
+import {OutputContribution} from '@theia/output/lib/browser/output-contribution';
 import {OutputChannel, OutputChannelManager} from '@theia/output/lib/common/output-channel';
 import {OutputChannelRegistryMain} from '../../api/plugin-api';
 
 export class OutputChannelRegistryMainImpl implements OutputChannelRegistryMain {
-    private delegate: OutputChannelManager;
+    private outputChannelManager: OutputChannelManager;
+    private viewContribution: { openView: (arg: { activate: boolean }) => Promise<OutputWidget> };
```

Comment:
`{ openView: (arg: { activate: boolean }) => Promise<OutputWidget> }` =>`OutputContribution `?

Reply:
Just to clarify what we can use. I have fixed it.